tests/src/ExistingSiteJavascript/CohesionTestBase.php

```diff
@@ -34,6 +34,9 @@ protected function setUp(): void {
    *   A wrapper object for interacting with the layout canvas.
    */
   protected function getLayoutCanvas(): LayoutCanvas {
+    /** @var \Drupal\FunctionalJavascriptTests\JSWebAssert $assertSession */
+    $assertSession = $this->assertSession();
+    $assertSession->waitForElementVisible('css', '.ssa-layout-canvas');
```

Comment:
@apathak18 I'm reverting this change, as the same is happening at line #40. If you still think, this is needed, let's discuss and create a separate PR.

Reply:
Yes this needs to be revert.